### DISCUSSION TITLE
How do you recommend chaining the API LLM General Link nodes together?

### DISCUSSION THREAD

**Discussion by Jonseed** [heshengtao/comfyui_LLM_party#116] [Q&A]
If we want to keep the conversation/memory, but use separate nodes for separate prompts, how do we do that? Does the history output connect to the system input of the next node? Is main_brain only turned on for the first node?

**Comment by heshengtao** [comment]
The memory of each LLM node is a JSON file and does not affect each other. When the main brain is false, the LLM node will become an LLM tool, which is equivalent to you using one LLM as another LLM tool.

**Comment by Jonseed** [comment]
Ok, so how do you connect multiple nodes together that use different prompts, but keep the conversation context?
